sdk/go/common/workspace/templates_zip_test.go

```diff
@@ -150,6 +150,18 @@ func TestRetrieveZIPTemplates(t *testing.T) {
 		if err != nil {
 			t.Errorf("Failed to write nested file to zip archive: %s", err)
 		}
+
+		if req.URL.Path != "/no-pulumi-yaml/foo.zip" {
+			fileHandle, err := writer.Create("Pulumi.yaml")
```

Comment:
This looks like it is adding a Pulumi.yaml?

Reply:
It is for the test handler. Right now the test http server doesn't contain a Pulumi.yaml in it's zip response

Reply:
But this path is "no-pulumi-yaml/" 😕

Reply:
🤦 ah right - good catch

Reply:
Ah right